packages/ag-grid-theme/css/salt-ag-theme.scss

```diff
@@ -363,6 +363,20 @@ $ag-theme-salt-params-touch-dark: map-merge(
   $ag-theme-salt-params-dark
 );
 
+.ag-theme-salt-touch-dark,
+.ag-theme-salt-low-dark,
+.ag-theme-salt-medium-dark,
+.ag-theme-salt-high-dark {
+  color-scheme: dark;
```

Comment:
Ah nice, I didn't know there was a native setting that we could also overwrite

Reply:
But what problem are we solving with this change?

Reply:
see description * MTK ticket